workshop_code/workshop_3/Workshop3.ipynb

```diff
@@ -0,0 +1,565 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2018-10-20T04:16:52.501732Z",
+     "start_time": "2018-10-20T04:16:52.314473Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Load dataset for linear regression, and inspect it\n",
+    "\n",
+    "### FILL IN ###\n",
+    "df_energy.head(5)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2018-10-20T04:16:53.283996Z",
+     "start_time": "2018-10-20T04:16:53.216224Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Remove date column, rv1, and rv2 columns \n",
+    "df_energy.drop('date', axis=1, inplace=True) # axis=1 to indicate we're dropping a column, inplace means to not make a copy\n",
+    "df_energy.drop('rv1', axis=1, inplace=True)\n",
+    "df_energy.drop('rv2', axis=1, inplace=True)\n",
+    "\n",
+    "# Inspect modified dataset\n",
+    "### FILL IN ###"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2018-10-20T04:16:54.019440Z",
+     "start_time": "2018-10-20T04:16:53.968849Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Sum appliances and lights for total energy\n",
+    "\n",
+    "### FILL IN ###\n",
+    "df_energy.head(5)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2018-10-20T04:16:55.050538Z",
+     "start_time": "2018-10-20T04:16:54.981300Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Then drop appliances and lights, as they are now redundant features\n",
+    "\n",
+    "df_energy.drop('Appliances', axis=1, inplace=True)\n",
+    "df_energy.drop('lights', axis=1, inplace=True)\n",
+    "df_energy.head(5)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2018-10-20T04:16:55.897261Z",
+     "start_time": "2018-10-20T04:16:55.848494Z"
+    },
+    "code_folding": []
+   },
+   "outputs": [],
+   "source": [
+    "# Split the dataset into features and labels for training, testing\n",
+    "\n",
+    "### FILL IN ###\n",
+    "df_energy_X.head(5)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2018-10-19T02:34:44.406540Z",
+     "start_time": "2018-10-19T02:34:44.393274Z"
+    },
+    "code_folding": []
+   },
+   "outputs": [],
+   "source": [
+    "df_energy_y.head(5)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2018-10-20T04:33:19.394285Z",
+     "start_time": "2018-10-20T04:33:19.068177Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Method that takes in dataframe, and degree of polynomial\n",
```

Comment:
I might just be really nit picky, but Python method comments are inside the function with ''' ''' :D